src/controlnet_aux/leres/pix2pix/util/get_data.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import print_function
```

Comment:
Do we need this file? There are some packages that we don't install by default like `bs4`